### PR TITLE
Linting ``nextcord``

### PR DESCRIPTION
**Pull request by VincentRPS**
## Summary

Linted nextcord to be more readable; And Fully follow pep8

## Checklist

- [x] If code changes were made then they have been tested.

### CODE REVIEW

**Comment by ooliver1** [comment]
mmmm not sure about this now, linting code you did not make can cause issues

File: .gitignore [hunk 1]
```diff
@@ -14,8 +14,12 @@ docs/crowdin.py
 *.jpg
 *.flac
 *.mo
-pyproject.toml
+.env
+*.ts
+tsconfig.json
 dist/
+pyproject.toml
+*mp4
```
Comment (ooliver1) [inline .gitignore:22]: like what are mp4's doing here

Comment (VincentRPS) [reply .gitignore:22]: @ooliver1 I remember there was a conversation to support mp4 or not in voice rec just added is Incase

Comment (ooliver1) [reply .gitignore:22]: Generally you don't find people storing the mp4's in the same dir as the repo, anyway if you're bothering to do that there's a lot more formats

Comment (ooliver1) [reply .gitignore:22]: Oh also keep PR's to one issue/topic maybe

Comment (VincentRPS) [reply .gitignore:22]: Ah Okay

File: .gitignore [hunk 2]
```diff
@@ -14,8 +14,9 @@ docs/crowdin.py
 *.jpg
 *.flac
 *.mo
-pyproject.toml
```
Comment (TAG-Epic) [inline .gitignore:17]: should not be changed?

File: aliasgen.py [hunk 3]
```diff
@@ -40,7 +40,7 @@
 
 def scan_dir(folder):
     for file in folder.glob("*"):
-        unprefixed = str(file)[len("nextcord") + 1:]
+        unprefixed = str(file)[len("nextcord") + 1 :]
```
Comment (TAG-Epic) [inline aliasgen.py:43]: this looks awful, are you sure this isn't a linter bug?

File: aliasgen.py [hunk 4]
```diff
@@ -79,7 +79,7 @@ def scan_dir(folder):
                 # __all__
                 module_all = getattr(module, "__all__", None)
                 if module_all is not None:
-                    module_all_quotes = ", ".join([f"\"{item}\"" for item in module_all])
+                    module_all_quotes = ", ".join([f'"{item}"' for item in module_all])
```
Comment (TAG-Epic) [inline aliasgen.py:82]: why replace this?

File: discord/abc.py [hunk 5]
```diff
@@ -27,5 +27,58 @@
 Autogenerated by aliasgen.py
 """
 
-from nextcord.abc import AllowedMentions, Any, Callable, ChannelType, ClientException, Connectable, Dict, File, GCH, GuildChannel, GuildSticker, HistoryIterator, InvalidArgument, Invite, List, MISSING, Messageable, Optional, PermissionOverwrite, Permissions, PrivateChannel, Protocol, Role, Sequence, Snowflake, StickerItem, T, TYPE_CHECKING, Tuple, TypeVar, Typing, Union, User, VoiceClient, VoiceProtocol, _Overwrites, _Undefined, _undefined, annotations, asyncio, copy, overload, runtime_checkable, utils
-__all__ = ("Snowflake", "User", "PrivateChannel", "GuildChannel", "Messageable", "Connectable")
\ No newline at end of file
+from nextcord.abc import (
```
Comment (TAG-Epic) [inline discord/abc.py:30]: aliasgen doesn't generate linted scripts? Maybe add that as a step?

File: docs/conf.py [hunk 6]
```diff
@@ -67,36 +67,38 @@
...
 # built documents.
 #
 # The short X.Y version.
 
-version = ''
-with open('../nextcord/__init__.py') as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE).group(1)
+version = ""
+with open("../nextcord/__init__.py") as f:
+    version = re.search(
```
Comment (TAG-Epic) [inline docs/conf.py:93]: this looks awful.

File: docs/conf.py [hunk 7]
```diff
@@ -18,45 +18,45 @@
...
-sys.path.insert(0, os.path.abspath('..'))
-sys.path.append(os.path.abspath('extensions'))
+sys.path.insert(0, os.path.abspath(".."))
+sys.path.append(os.path.abspath("extensions"))
 
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+# needs_sphinx = '1.0'
```
Comment (TAG-Epic) [inline docs/conf.py:27]: comments are half linted here where, it uses single instead of double quotes in comments?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
I honestly think we should wait until voice receive and slash commands are merged then make a linting CI and setup.py task imo

File: discord/abc.py (hunk 5 above)
Comment (VincentRPS) [reply discord/abc.py:30]: hm?

**Comment by VincentRPS** [comment]
actually i agree with epic to make this a cd/ci process so i am gonna close this

**Pull request closed without merging**